pkg/cluster/k8sres.go

```diff
@@ -1048,11 +1048,13 @@ func (c *Cluster) getNumberOfInstances(spec *acidv1.PostgresSpec) int32 {
 	cur := spec.NumberOfInstances
 	newcur := cur
 
-	/* Limit the max number of pods to one, if this is standby-cluster */
 	if spec.StandbyCluster != nil {
-		c.logger.Info("Standby cluster can have maximum of 1 pod")
-		min = 1
-		max = 1
+		if newcur > 1 {
+			c.logger.Warningf("operator only supports standby clusters with 1 pod")
```

Comment:
better to mention credentials's problem in the log message for readability and comment the code

Reply:
that's another story and not related to any pod count. There will be enough error logs already to notice and with improved docs, people now know what to do.